sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/StatusCode.java

```diff
@@ -17,20 +17,81 @@
 package kalix.javasdk;
 
 /**
- * Interface used to represent a status code, typically used when replying with an error. **NOT**
- * for user extension.
+ * Interface used to represent HTTP status code. **NOT** for user extension.
```

Comment:
can we sealed it to avoid user extension?

Reply:
tried it.. but I think we are still in java 11 here, right?